README.md

```diff
@@ -1,2 +1,42 @@
 # Mbed OS Examples for the ATECC608A
 
+This repository contains examples that demonstrate how to use the ATECC608A secure element with Mbed OS. The `atecc608a` example demonstrates use of the ATECC608A with Mbed Crypto. Additional examples of how to generate a certificate using pre-provisioned ATECC608A keys and how to use ATECC608A with Mbed TLS are yet to come.
+
+## Prerequisites
+
+* [Install Mbed CLI](https://os.mbed.com/docs/mbed-os/latest/tools/installation-and-setup.html).
+
+* [Install the arm-none-eabi-ggc toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads).
+
+## Building the ATECC608A example
+
+```sh
+git clone git@github.com:ARMmbed/mbed-os-example-atecc608a.git
+cd mbed-os-example-atecc608a/
+source ~/venvs/mbed/bin/activate
```

Comment:
Do you need this line if you don't set up a virtual env?

Reply:
No, we don't need this line if we don't set up virtual environments. I don't think other examples mention virtual environments, so probably all mentioned of virtual environments should go away.